extensions/wikia/MiniEditor/js/MiniEditor.js

```diff
@@ -234,6 +234,26 @@ var MiniEditor = {
 				startupFocus: !preloading
 			}, options.config), event);
 
+			// VOLDEV-25: Provide LinkSuggest support if necessary
+			if (window.wgEnableMiniEditorLinkSuggest) {
+				if (this.config.editorSuite === 'ckeditor') {
+					// RTE
+					wikiaEditor.on('ck-mode', function () {
+						if (wikiaEditor.ck.mode !== 'wysiwyg') {
+							// load the module only if the user actually switches to source
+							mw.loader.using('ext.wikia.LinkSuggest', function() {
+								$wrapper.find( 'textarea.cke_source' ).linksuggest();
+							});
+						}
+					});
+				} else {
+					// mweditor
+					mw.loader.using('ext.wikia.LinkSuggest', function() {
```

Comment:
This line (```mw.loader.using```) is repeated twice in this area - please consider creating a small (local) function that just takes selector of where of initialize linksuggest. So then you would call it like ```initLinkSuggest('textarea.cke_source')``` and ```initLinkSuggest('#' + wikiaEditor.instanceId)```